go/libraries/doltcore/schema/schema.go

```diff
@@ -148,6 +148,16 @@ func SchemasAreEqual(sch1, sch2 Schema) bool {
 	if !colCollIsEqual {
 		return false
 	}
+
```

Comment:
do we still need the check on 147 if we're doing these both below?

Reply:
Yes. You can have two different orderings for the schema definition that have the same key and value orderings.

```sql
CREATE table A (
    pk int PRIMARY KEY,
    col1 int,
    col2 int
);

CREATE table B (
    col1 int,
    pk int PRIMARY KEY,
    col2 int,
);

-- Should the schema for table A equal table B?
```